pkg/apis/litmuschaos/v1alpha1/chaosexperiment_types.go

```diff
@@ -72,6 +72,8 @@ type ExperimentDef struct {
 	ConfigMaps []ConfigMap `json:"configmaps,omitempty"`
 	// Secrets contains a list of Secrets
 	Secrets []Secret `json:"secrets,omitempty"`
+	// Annotations that needs to be provided in the pod for pod that is getting created
+	AppAnnotations []ApplicationAnnotations `json:"appannotations"`
```

Comment:
We shall be defining its own var ExperimentAnnotations over reusing the global struct AppAnnotations. The idea, as started in the issue comments is that the annotation here happens to be a property inside experimentDefinition which we may be overriding with an instance-specific value from chaosengine. (ChaosExperiment = Static definition / ChaosEngine = instance basis) 

```
ExperimentAnnotations map[string]string `json:"experimentannotations,omitempty"`
```